XMonad/Hooks/DynamicProperty.hs

```diff
@@ -1,113 +1,12 @@
------------------------------------------------------------------------------
 -- |
 -- Module      :  XMonad.Hooks.DynamicProperty
 -- Description :  Apply a ManageHook to an already-mapped window.
 -- Copyright   :  (c) Brandon S Allbery, 2015
 -- License     :  BSD3-style (see LICENSE)
---
 -- Maintainer  :  allbery.b@gmail.com
--- Stability   :  unstable
--- Portability :  not portable
---
--- Module to apply a ManageHook to an already-mapped window when a property
--- changes. This would commonly be used to match browser windows by title,
--- since the final title will only be set after (a) the window is mapped,
--- (b) its document has been loaded, (c) all load-time scripts have run.
--- (Don't blame browsers for this; it's inherent in HTML and the DOM. And
--- changing title dynamically is explicitly permitted by ICCCM and EWMH;
--- you don't really want to have your editor window umapped/remapped to
--- show the current document and modified state in the titlebar, do you?)
---
--- This is a handleEventHook that triggers on a PropertyChange event. It
--- currently ignores properties being removed, in part because you can't
--- do anything useful in a ManageHook involving nonexistence of a property.
 --
--- This module could also be useful for Electron applications like Spotify
--- which sets its WM_CLASS too late for window manager to map it properly.
---
------------------------------------------------------------------------------
-
-module XMonad.Hooks.DynamicProperty ( -- * Usage
-                                      -- $usage
-
-                                      -- * Documentation
-                                      dynamicPropertyChange
-                                    , dynamicTitle
+module XMonad.Hooks.DynamicProperty {-# DEPRECATED "Use XMonad.Hooks.OnPropertyChange instead." #-}
+                                    ( module XMonad.Hooks.OnPropertyChange
```

Comment:
This is a good thing in general, I think (thanks!), but we should at least have wrappers for `dynamicPropertyChange` and `dynamicTitle` here, as their names have changed

Reply:
Absolutely! I didn't have time to look :-)